corehq/motech/repeaters/models.py

```diff
@@ -1422,7 +1422,7 @@ def has_failed(record):
 def format_response(response) -> Optional[str]:
     if not is_response(response):
         return None
-    response_text = getattr(response, "text", "")[:ONE_MB]
+    response_text = getattr(response, "text", "")[:MAX_REQUEST_LOG_LENGTH]
```

Comment:
This is only used for RepeatRecordAttempt messages, which appear in the Repeat Records report. We can make this text _much_ shorter, because it is not used for inspecting payloads. (I also don't think we ever delete RepeatRecordAttempts!)